src/components/QueriedSponsorshipsTable.tsx

```diff
@@ -59,16 +61,20 @@ export function QueriedSponsorshipsTable({
                 orderDirection={orderDirection}
                 onOrderChange={onOrderChange}
                 columns={[
-                    {
-                        displayName: 'Stream ID',
-                        valueMapper: ({ streamId }) => (
-                            <StreamIdCell streamId={streamId} />
-                        ),
-                        align: 'start',
-                        isSticky: true,
-                        key: 'streamInfo',
-                        sortable: true,
-                    },
+                    ...(hideStreamId == false
```

Comment:
I guess that's the way for now.

Too bad the table component doesn't support nullish columns (null = skip).

Reply:
Yeah, let's improve ScrollTable later.